### PR TITLE
Add a note to generated systemd proxy files

### PR DESCRIPTION
**Pull request by wupeka**
Add a comment in juju proxy systemd config file on how to enable systemd to use it.

### CODE REVIEW

File: proxy/proxy.go [hunk 1]
```diff
@@ -92,7 +92,10 @@ func (s *Settings) AsEnvironmentValues() []string {
 // DefaultEnvironment="http_proxy=...." "HTTP_PROXY=..." ...
 func (s *Settings) AsSystemdDefaultEnv() string {
 	lines := s.AsEnvironmentValues()
-	rv := `[Manager]
+	rv := `# To make juju control global systemd proxy settings create
```
Comment (rogpeppe) [inline proxy/proxy.go]: I'm finding this a bit hard to parse.

Perhaps:

    # To allow juju to control the global systemd proxy settings,
    # create symbolic links to this file from within /etc/systemd/system.conf.d/
    # and /etc/systemd/users.conf.d/.


?

**Comment by wupeka** [comment]
$$merge$$

**Comment by wupeka** [comment]
$$merge$$
